### ISSUE TITLE
Respect slow animations in the iOS simulator

### ISSUE THREAD

**Issue by ZevEisenberg** [AntonTheDev/FlightAnimator#19]
It probably won't be easy, but this may help: http://stackoverflow.com/questions/13275294/detect-if-slow-animations-is-on-off-in-ios-simulator-in-code

**Comment by AntonTheDev** [comment]
This one is definitely gonna throw me for a loop to ensure I can pass through apple's approval system

**Comment by ZevEisenberg** [comment]
Only do it [if you're building for simulator](http://stackoverflow.com/a/24869607/255489). The code won't be compiled in to device builds.

Related: it could be cool to have a global FlightAnimator override for slow animations, in case you want more control over them than the simulator can offer.

**Comment by AntonTheDev** [comment]
Like a multiply the duration by 10x type of thing?

**Comment by ZevEisenberg** [comment]
Yep! It's something I've often wished I'd built in to my own animation code, but I never remember to do it until it's impractically late.

**Comment by AntonTheDev** [comment]
I think I might implement it as part of the logging solution I'm setting up at the moment. When debug mode is enable I'll add the the ability for a shake to turn on/off 10x timers while logging is turned on for debug
